docs/ci-proposals.md

```diff
@@ -4,8 +4,6 @@ It's a best practice to propose deployments from a CI process instead of using t
 
 This guide will show you how to integrate proposals into your CI process. We'll create two workflows: one that dry runs the proposal when a pull request is opened or updated, and another that proposes the deployment when the pull request is merged.
 
-We'll use GitHub Actions as the CI platform in this guide. You can still follow this guide if you're using a different CI platform, but the exact configuration may be slightly different.
```

Comment:
Should we mention up-front that this guide only works for GitHub actions?

Reply:
We can if you want, I'm not that worried about it.

Reply:
Let's not bother then